### PR TITLE
Only put the block header chain in the indent guide

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
It now looks like this:

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
image

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

File: src/EditorFeatures/Next/Structure/BlockContextProvider.cs [hunk 1]
```diff
@@ -50,23 +73,119 @@ public void Dispose()
...
+                return result;
+            }
+
+            private IWpfTextView CreateElisionBufferView(ITextBuffer finalBuffer)
+            {
+                return RoslynOutliningRegionTag.CreateShrunkenTextView(
+                    _provider._textEditorFactoryService, finalBuffer);
+            }
+
+            private ITextBuffer CreateProjectionBufferForBlockHeaders()
```
Comment (tmeschter) [inline src/EditorFeatures/Next/Structure/BlockContextProvider.cs]: Any unit tests for this?

Comment (CyrusNajmabadi) [inline src/EditorFeatures/Next/Structure/BlockContextProvider.cs]: No.  But i will attempt to add one.  In general, we're not great with very high level UI tests.

File: src/EditorFeatures/Next/Structure/BlockContextProvider.cs [hunk 2]
```diff
@@ -50,23 +73,119 @@ public void Dispose()
...
+                // We want to create a projection buffer that will show the block tags like so:
+                //
+                //      namespace N
+                //          class C
+                //              public void M()
+                //
+                // To do this, we grab the 'header' part of each block span and stich them 
+                // all together one after the other.
+                //
+                // We consider hte 'header' to be from the start of the line containing the
```
Comment (dpoeschl) [inline src/EditorFeatures/Next/Structure/BlockContextProvider.cs]: hte -> the

**Pull request merged**